spec/predicates/runtime-trace.md

```diff
@@ -4,37 +4,59 @@ Type URI: https://in-toto.io/runtime-trace/v0.1.0
 
 Version: 0.1.0
 
+Authors: Parth Patel (@pxp928), Shripad Nadgowda (@nadgowdas),
+  Aditya Sirish A Yelgundhalli (@adityasaky)
+
 ## Purpose
 
-This predicate is used to describe system events that were part of the artifact build process. Artifact build processes today are complex and opaque with little to no insights into build actions. However, there are emerging introspection techniques that now allow us to monitor and capture system event logs during the build.
+This predicate is used to describe system events that were part of some software
+supply chain process, for example, the build process of an artifact. Supply
+chain processes today are complex and opaque with little to no insights into
+underlying system operations. However, there are emerging introspection
+techniques that now allow us to monitor and capture system event logs during the
+target process.
+
+## Use Cases
+
+Generally, this predicate can be used with a runtime trace of any process.
+However, the primary use case driving the development of this predicate is its
+applicability to runtime traces of build processes.
```

Comment:
Several of the fields in the predicate seem to imply that the "process" your tracing is more specifically a CI/CD or some other scripted process, versus a discreet manually entered CLI command? I think it would be helpful to just add this extra small detail in the use cases (or even purpose) description to be crystal clear.

Reply:
Done!